src/Exception/SerializableException.php

```diff
@@ -68,23 +68,23 @@ public function getLine()
     /**
      * @return array
      */
-    public function getTrace()
+    public function getTrace(): array
     {
         return $this->data['trace'];
     }
 
     /**
      * @return string
      */
-    public function getTraceAsString()
+    public function getTraceAsString(): string
     {
         return implode("\n", $this->data['trace']);
     }
 
     /**
      * @return string
      */
-    public function getMessage()
+    public function getMessage(): string
```

Comment:
Please do not add native php typehints in this PR as this introduces a BC break. 
Focus on PHP 8.0 compatibility only.
Thanks!